### PR TITLE
Infinitude of Primes theorem

### PR DESCRIPTION
**Pull request by HoanNguyen92**
Dear professor Thomas Hales,

I am trying to complete "Infinitude of Primes theorem". Could you have a look and give me some comments?

Best regards,
Hoan.

### CODE REVIEW

**Comment by jesse-michael-han** [comment]
Hi Hoan, currently the Lean syntax extension for Sphinx is incompatible with reST code blocks started with double colons, e.g.

```preformally ::```
 ```    code goes here```

Instead, for pseudocode you should use the directive `.. code-block:: text`, and for Lean code you should use the directive `.. code-block:: lean`.

**Comment by HoanNguyen92** [comment]
Hi Jesse,
I edited as you said. Could you have a look?

**Comment by jesse-michael-han** [comment]
You've added the `.. code-block::` directives, which is good. However, there are still some issues when one tries to compile with the lean_sphinx extension. I'll go through your code and point out where the issues are.

File: sphinx/Infinitude_of_Primes.rst [hunk 1]
```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
 description, see
 `Infinitude of Primes <https://en.wikipedia.org/wiki/Prime_number#Infiniteness>`_.
 
-Informally statement
```
Comment (jesse-michael-han) [inline sphinx/Infinitude_of_Primes.rst]: This is still formatted as a reST code block, which will not compile. The informal statement of the theorem can be done with plain text, e.g. "The theorem states that there are infinitely many primes."

File: sphinx/Infinitude_of_Primes.rst [hunk 2]
```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
...
+Informally statement ::
 
   There are infinitely many :ref:`prime` numbers.
 
 
 Preformally ::
```
Comment (jesse-michael-han) [inline sphinx/Infinitude_of_Primes.rst]: `Preformally ::` (and `Formally statement ::` below) still use the double-colon syntax for reST code blocks. Now that you've inserted the `.. code-block::` directives, you can get rid of the double colons and just treat them as subsection headers.

File: sphinx/Infinitude_of_Primes.rst [hunk 3]
```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
...
+  ..
code-block::text
 
-  Prime number
+    Prime number
```
Comment (jesse-michael-han) [inline sphinx/Infinitude_of_Primes.rst]: The "preformalization" should just be a pseudocode version of the formal abstract, and in fact can be removed once you finish writing the formal abstract in Lean.

However, you have the right idea with spelling out the concepts you need to state this formal abstract. Maybe you could create definition pages for "prime" and "infinite set"?

Comment (HoanNguyen92) [reply sphinx/Infinitude_of_Primes.rst]: You mean I should create two files ".rst" for "prime" and "infinite set"?

**Comment by HoanNguyen92** [comment]
Hi Jesse,

I tried to fix the issues you said. Could you have a look?

**Comment by jesse-michael-han** [comment]
OK, merged; I just fixed some minor grammar errors (e.g. "informally" is an adverb, but when we want to modify the noun "statement", we want the adjective "informal" instead.)

**Comment by HoanNguyen92** [comment]
OK, thank you! :)

**Pull request merged**